### PR TITLE
[Toolchain] Add description for password requirement

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit adds password requirement description by using InputBoxOptions' placeHolder field.

ONE-vscode-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: src/Toolchain/ToolchainEnv.ts [hunk 1]
```diff
@@ -71,8 +71,10 @@ class Env implements BuilderJob {
     const rootJobs = this.workFlow.jobs.filter((j) => j.root === true);
     if (rootJobs.length > 0) {
       Logger.info(this.logTag, "Showing password prompt");
+      const helpMsg: string =
+        "It requires sudo authenticataion. Please enter your password.";
```
Comment (dayo09) [inline src/Toolchain/ToolchainEnv.ts:75]:
```suggestion
        "It requires sudo authentication. Please enter your password.";
```

Comment (jyoungyun) [reply src/Toolchain/ToolchainEnv.ts:75]: How about using `Enter password for blah blah` help message?
The vscode basically shows a help message like this.
a

Thank you for your contribution! :)

Comment (hseok-oh) [reply src/Toolchain/ToolchainEnv.ts:75]: Then let's use "Enter password for sudo authentication". (`Enter password for` from vscode, `sudo authentication` from ubuntu software updater`)

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]

**Pull request merged**